### DISCUSSION TITLE
Self-Hosted ChromaDB on Langflow

### DISCUSSION THREAD

**Discussion by federicobiamonte** [langflow-ai/langflow#5250] [Q&A]
Hi, 

I am currently experimenting with Langflow and I am trying to connect to an Enterprise ChromaDB to create a RAG.
I am currently having issues with connecting to my service. Is anyone experiencing the same issue?

If so, have you found any solution?
Thanks

**Comment by nrblock** [comment]
I think the component is broken. I can connect to my ChromaDB fine through a client in python, and confirmed that I see a heartbeat too from the machine running Langflow, but it will not connect to the server and gives a message indicating that it worked.
